lib/models.ts

```
@@ -0,0 +1,146 @@
+    switch (fallbackProvider) {
+      case "google":
+        mainModel = google(defaultModelId, {
+          useSearchGrounding: useWebSearch,
+        });
+        break;
+      case "azure":
+        const azure = createAzure({
+          baseURL: process.env.AZURE_BASE_URL,
+          apiKey: process.env.AZURE_API_KEY,
+          apiVersion: "2023-03-15-preview",
+        });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)